src/tests/testData/mockData.json

```diff
@@ -69,5 +69,149 @@
         }
       }
     }
+  ],
```

Comment:
@DancingAxolotl Good idea, we should migrate all RPC mocks to this approach in future